### ISSUE TITLE
Banner - Color contrast accessibility issue

### ISSUE THREAD

**Issue by daminihs** [wordplaydev/wordplay#672]
### What happened?

![Image](https://github.com/user-attachments/assets/20b8ce9c-a673-41d6-b4a8-ce05009cee53)

Tutorial page - Evaluate 15/21
In this banner "This editing mode is experimental. Discuss Improvements", the word "Discuss" has insufficient color contrast of 1.65 (foreground color: #ffb000, background color: #fe6100, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1.
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows

**Comment by amyjko** [comment]
Good catch, thanks @daminihs!
